.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
```

Comment:
lets add most commonly used styleguides - you can choose airbnb style or google
https://www.npmjs.com/package/eslint-config-airbnb
https://www.npmjs.com/search?q=eslint-config%20google

Note those are plain js styles so you need to make them support TS